lib/mongo/database.rb

```diff
@@ -129,27 +139,44 @@ def collection_names(options = {})
     # @param [ Hash ] options
     #
     # @option options [ Hash ] :filter A filter on the collections returned.
+    # @option options [ true, false ] :name_only Indicates whether command
+    #   should return just collection/view names and type or return both the
+    #   name and other information
+    # @option options [ true, false ] :authorized_collections A flag, when
+    #   set to true and used with nameOnly: true, that allows a user without the
+    #   required privilege to run the command when access control is enforced
+    #
     #   See https://docs.mongodb.com/manual/reference/command/listCollections/
     #   for more information and usage.
     #
     # @return [ Array<Hash> ] Array of information hashes, one for each
     #   collection in the database.
     #
     # @since 2.0.5
-    def list_collections(**options)
-      View.new(self).list_collections(**options)
+    def list_collections(options = {})
+      View.new(self).list_collections(options)
     end
 
     # Get all the non-system collections that belong to this database.
     #
     # @note The set of returned collections depends on the version of
     #   MongoDB server that fulfills the request.
     #
+    # @param [ Hash ] options
+    #
+    # @option options [ Hash ] :filter A filter on the collections returned. 
+    # @option options [ true, false ] :authorized_collections A flag, when
+    #   set to true and used with nameOnly: true, that allows a user without the
```

Comment:
`name_only` please, since users will be specifying the option using Ruby syntax.